src/core/api/public_api.ts

```diff
@@ -1714,12 +1714,17 @@ class Player extends EventEmitter<IPublicAPIEvent> {
 
   /**
    * @returns {Array.<Object>|null}
+   * @deprecated
    */
   getImageTrackData() : IBifThumbnail[] | null {
+    warnOnce("`getImageTrackData` is deprecated." +
```

Comment:
I want a public debate about the warnOnce

Reply:
It was added to avoid spamming the user. We don't know how often he uses this API. Warning him once about deprecation seems enough, the API still works.

There would also be a chance of `console.warn` calls to make their way to production.